website/content/en/docs/cli/operator-sdk_olm_install.md

```diff
@@ -19,7 +19,7 @@ operator-sdk olm install [flags]
   -h, --help                   help for install
       --olm-namespace string   namespace where OLM is to be installed. (default "olm")
       --timeout duration       time to wait for the command to complete before failing (default 2m0s)
-      --version string         version of OLM resources to install (default "latest")
+      --version string         version of OLM resources to install; non-defaultversions are downloaded from GitHub.
```

Comment:
It is not very clear for me. 
```suggestion
      --version string         version of OLM resources to install. Note that, if this value be informed then the installation will be made via the manifests which are downloaded from the OLM repository E.g <add example>.
```

Reply:
I'd rather keep this short:
```suggestion
      --version string         version of OLM resources to install (from a Github release); if not set, a pre-packaged OLM version will be installed
```
Also we can set this default using the internally-set OLM version.